packages/web-api/src/types/request/chat.ts

```diff
@@ -218,12 +218,7 @@ export type ChatScheduledMessagesListArguments = OptionalArgument<
     Partial<Channel>
 >;
 
-export interface ChatStartStreamArguments
-  extends TokenOverridable,
-    Channel,
-    Partial<ThreadTS>,
-    Partial<MarkdownText>,
-    Unfurls {
+export interface ChatStartStreamArguments extends TokenOverridable, Channel, Partial<MarkdownText>, ThreadTS, Unfurls {
```

Comment:
note: `npm test` linter requires this to be a single-line. I ordered it alphabetically.